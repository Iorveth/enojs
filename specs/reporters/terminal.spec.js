const context = require('./contextFixture.js');
const report = require('../../lib/reporters/report.js');

describe('Terminal reporter', () => {
  it('produces colored terminal output', () => {
    context.reporter = 'terminal';

    const snippet = report(
      context,
      context.instructions[1],
      context.instructions[0]
    )

    // Uncomment this to inspect the snippet correctness in a terminal for review
    // console.log(snippet);

    expect(snippet).toMatchSnapshot();
  });
});
