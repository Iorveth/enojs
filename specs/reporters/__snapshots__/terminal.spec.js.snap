// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terminal reporter produces colored terminal output 1`] = `
"[1m   Line | Content[0m
[33m *    1 | > comment[0m
[41m >    2 | language: eno[0m
      3 | 
      4 | 
[1m   ...[0m
"
`;
